concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-900	900	König, Königin, deren Sohn	Prinz	prince	2554	PRINCE (RULER)
+Steinthal-1875-1565-901	901	freier Mann	freier Mann	free man	384	FREEMAN
+Steinthal-1875-1565-902	902	berathende Versammlung	berathende Versammlung	gathering		
+Steinthal-1875-1565-903	903	Sklave	Sklave	male slave	812	SLAVE
+Steinthal-1875-1565-904	904	Sklavin	Sklavin	female slave	812	SLAVE
+Steinthal-1875-1565-905	905	befehlen	befehlen	to order	1128	ORDER
+Steinthal-1875-1565-906	906	gehorchen	gehorchen	to obey	1635	OBEY
+Steinthal-1875-1565-907	907	gehorsam	gehorsam	obedient		
+Steinthal-1875-1565-908	908	Priester	Priester	priest	852	PRIEST
+Steinthal-1875-1565-909	909	Zauberer (Arzt)	Zauberer	sorcerer	823	SORCERER
+Steinthal-1875-1565-910	910	Zauberer (Arzt)	Arzt	doctor	597	PHYSICIAN
+Steinthal-1875-1565-911	911	Arznei	Arznei	medicine	1372	MEDICINE
+Steinthal-1875-1565-912	912	Opfer	Opfer	sacrifice	1103	SACRIFICE
+Steinthal-1875-1565-913	913	beten (Ceremonien beim Beten)	beten	to pray	24	PRAY
+Steinthal-1875-1565-914	914	beten (Ceremonien beim Beten)	Ceremonien beim Beten	ceremonies while praying		
+Steinthal-1875-1565-915	915	fasten	fasten	to fast	303	ABSTAIN FROM FOOD
+Steinthal-1875-1565-916	916	schwören	schwören	to swear	1939	SWEAR
+Steinthal-1875-1565-917	917	Geister beschwören	Geister beschwören	to conjure spirits		
+Steinthal-1875-1565-918	918	fluchen	fluchen	to curse	8	CURSE
+Steinthal-1875-1565-919	919	segnen	segnen	to bless	391	BLESS
+Steinthal-1875-1565-920	920	lesen	lesen	to read	724	READ
+Steinthal-1875-1565-921	921	schreiben	schreiben	to write	1672	WRITE
+Steinthal-1875-1565-922	922	abschreiben	abschreiben	to transcribe		
+Steinthal-1875-1565-923	923	Todtengräber	Todtengräber	gravedigger		
+Steinthal-1875-1565-924	924	Klageweiber	Klageweiber	wailers, weepers		
+Steinthal-1875-1565-925	925	Schmied	Schmied	blacksmith	1001	BLACKSMITH
+Steinthal-1875-1565-926	926	Hirt	Hirt	shepherd	2544	SHEPHERD
+Steinthal-1875-1565-927	927	Jäger	Jäger	hunter	1053	HUNTER
+Steinthal-1875-1565-928	928	Fischer	Fischer	fisherman	1858	FISHERMAN
+Steinthal-1875-1565-929	929	Schiffer	Schiffer	skipper		
+Steinthal-1875-1565-930	930	Ackerbauer	Ackerbauer	farmer	979	FARMER
+Steinthal-1875-1565-931	931	Handwerker	Handwerker	craftsman		
+Steinthal-1875-1565-932	932	Kaufmann	Kaufmann	merchant	1082	MERCHANT
+Steinthal-1875-1565-933	933	Wirth	Wirth	innkeeper		
+Steinthal-1875-1565-934	934	Gast	Gast	guest	1782	GUEST
+Steinthal-1875-1565-935	935	Freund	Freund	friend	1325	FRIEND
+Steinthal-1875-1565-936	936	Feind	Feind	enemy	1358	ENEMY
+Steinthal-1875-1565-937	937	Nachbar	Nachbar	neighbour	1332	NEIGHBOUR
+Steinthal-1875-1565-938	938	Landsmann (Einheimischer)	Landsmann	fellow countryman		
+Steinthal-1875-1565-939	939	Landsmann (Einheimischer)	Einheimischer	native, local resident		
+Steinthal-1875-1565-940	940	Fremder	Fremder	foreigner	791	STRANGER
+Steinthal-1875-1565-941	941	Europäer	Europäer	European		
```

Comment:
Perhaps WHITE MAN?

Reply:
I think that would be too broad, especially since the list was intended for global use.